pyasic/miners/backends/marathon.py

```diff
@@ -302,3 +307,32 @@ async def _get_wattage_limit(
                 return web_miner_config["mode"]["concorde"]["power-target"]
             except LookupError:
                 pass
+
+    async def _get_pools(self, web_pools: list = None) -> List[PoolMetrics]:
+        if web_pools is None:
+            try:
+                web_pools = await self.web.pools()
+            except APIError:
+                pass
+
+        pools_data = []
+        if web_pools is not None:
+            try:
+                for pool_info in web_pools:
+                    url = pool_info.get("url")
+                    pool_url = PoolUrl.from_str(url) if url else None
+                    pool_data = PoolMetrics(
+                        accepted=pool_info.get("accepted"),
+                        rejected=pool_info.get("rejected"),
+                        get_failures=pool_info.get("stale"),
+                        remote_failures=pool_info.get("discarded"),
+                        active=pool_info.get("active", False),
```

Comment:
This key doesn't exist.  Could try finding the pool with highest priority that is alive, then when looping check if the index is the same as the active pool.